operating/manage-user-access/authentication-provider-structure.adoc

```diff
@@ -0,0 +1,79 @@
+:_content-type: ASSEMBLY
+[id="authentication-provider-structure"]
+= Authentication provider structure
+include::modules/common-attributes.adoc[]
+:context: authentication-provider-structure
+
+toc::[]
+
+[role="_abstract"]
+
+Authentication provider connects to third-party source of user identity(identity provider, or IdP). Authentication provider gets user identity from IdP, issues token based on this identity and returns token to the caller. Mentioned token is then sent within UI or API calls and allows ACS to authorize user.
+
+You can set up your own IdP right after provisioning using one of the guides under `Operating -> Managing user access`.
+
+[id="auth-provider-claim-mappings"]
+== Claim mappings
+Claim mappings allow customer to specify if attribute in user identity obtained from IdP should be added to ACS-issued token and with which attribute name.
+Example of claim mapping: from `roles` in user identity to `groups` in ACS-issued token.
+There are different default claim mappings for each auth provider type.
+
+[id="auth-provider-oidc-default-claim-mappings"]
+=== OIDC default claim mappings
+* `sub` to `userid`
+* `name` to `name`
+* `email` to `email`
+* `groups` to `groups`
+
+[id="auth-provider-auth0-default-claim-mappings"]
+=== Auth0 default claim mappings
+Same as OIDC default claim mappings.
+
+[id="auth-provider-saml20-default-claim-mappings"]
+=== SAML 2.0 default claim mappings
+* `Subject.NameID` to `userid`
+* every SAML `AttributeStatement.Attribute` from response gets mapped to its name
+
+[id="auth-provider-google-iap-default-claim-mappings"]
+=== Google IAP default claim mappings
+* `sub` to `userid`
+* `email` to `email`
+* `hd` to `hd`
+* `google.access_levels` to `access_levels`
+
+[id="auth-provider-userpki-default-claim-mappings"]
+=== User Certificates default claim mappings
+User Certificates is different from all other authentication providers in a sense that instead of communicating with a third-party IdP, it get user information from certificate used by user.
```

Comment:
```suggestion
User certificates differ from all other authentication providers because instead of communicating with a third-party IdP, they get user information from certificates used by the user.
```